ui-e2e-tests/custom-emojis.spec.ts

```diff
@@ -1,6 +1,13 @@
 import { test, expect } from '@playwright/test'
 
 test('View of custom emojis in an toots author display name', async ({ page, browserName }) => {
+	test.skip(
```

Comment:
qwik went from this human readable format:
![Screenshot 2023-02-17 at 17 07 01](https://user-images.githubusercontent.com/61631103/219719228-7427ec96-fe72-46c7-92ce-83463d247e79.png)

to this:
![Screenshot 2023-02-17 at 16 52 58](https://user-images.githubusercontent.com/61631103/219719352-92ee03f3-0f93-487e-bcd0-96268e0af838.png)

so I am not really sure how to deal with it and if mocking the emojis is at all possible now
😢

Reply:
I would open an issue with them to discuss.

Reply:
I've contacted them directly on discord 👍  (if they don't reply I'll open an issue on github 🙂)